docs/001_develop/03_client-capabilities/005_grids/003_grid-pro/index.mdx

```diff
@@ -2,17 +2,20 @@
 title: 'Grid Pro'
 sidebar_label: 'Grid Pro'
 id: grid-pro
-keywords: [genesis, foundation, ui, grid, grid-pro, ag]
+keywords: [genesis, foundation, ui, simples-data grid, grid-pro, ag]
 tags:
-  - connected
-  - datasource
+  - simple-data
   - grid
   - grid-pro
   - ag
 pagination_prev: develop/client-capabilities/grids/entity-manager/entity-manager
-pagination_next: develop/client-capabilities/grids/grid-tabulator/grid-tabulator
+pagination_next: develop/client-capabilities/grids/grid-pro/grid-pro-datasources
 ---
 
+import GridProExample from './grid-pro_example.js'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Grid Pro
 
 The `@genesislcap/grid-pro` package provides a collection of grid-related components and utilities for Genesis applications.
```

Comment:
Do we need to mention it wraps `ag-grid` here considering that we externally link to it multiple times?